core-services/prow/02_config/openshift/contour-operator/_pluginconfig.yaml

```diff
@@ -0,0 +1,16 @@
+approve:
```

Comment:
I think that we need to add `_prowconfig.yaml` file too. Because as we saw it with @sgreene570 without it Tide is not enabled in the repository and the PRs cannot be merged.
Don't know why it was not generated by `new-repo` target but I had to add it manually for `external-dns-operator`: https://github.com/openshift/release/pull/19766/files

Reply:
Thanks for the heads up on this one Andrey